### ISSUE TITLE
[Doc]: Is Qwen2-VL-72B supported?

### ISSUE THREAD

**Issue by pseudotensor** [vllm-project/vllm#8682]
### 📚 The doc issue

Unclear.

### Before submitting a new issue...

- [X] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by mgoin** [comment]
Qwen/Qwen2-VL-7B-Instruct has been tested and is used in an example: https://github.com/vllm-project/vllm/blob/0f961b3ce9ac3d3fd13e201c4358884bc094905e/examples/offline_inference_vision_language.py#L183

I would expect the larger version to work as well.

**Comment by pseudotensor** [comment]
```
docker pull vllm/vllm-openai:latest
docker stop qwen72bvll ; docker remove qwen72bvll
docker run -d --restart=always \
    --runtime=nvidia \
    --gpus '"device=0,1,2,3"' \
    --shm-size=10.24gb \
    -p 5000:5000 \
        -e NCCL_IGNORE_DISABLED_P2P=1 \
    -v /etc/passwd:/etc/passwd:ro \
    -v /etc/group:/etc/group:ro \
    -u `id -u`:`id -g` \
    -e HUGGING_FACE_HUB_TOKEN=$HUGGING_FACE_HUB_TOKEN \
    -v "${HOME}"/.cache:$HOME/.cache/ -v "${HOME}"/.config:$HOME/.config/   -v "${HOME}"/.triton:$HOME/.triton/  \
    --network host \
    --name qwen72bvll \
    gcr.io/vorvan/h2oai/h2ogpte-vllm:0.5.5-8714686a \
        --port=5000 \
        --host=0.0.0.0 \
        --model=Qwen/Qwen2-VL-72B-Instruct \
        --tensor-parallel-size=4 \
        --seed 1234 \
        --trust-remote-code \
        --max-model-len=32768 \
        --max-num-batched-tokens 65536 \
        --max-log-len=100 \
        --download-dir=$HOME/.cache/huggingface/hub &>> logs.vllm_server.qwen72bvll.txt

```

leads to:

![image](https://github.com/user-attachments/assets/3832c5b6-4dca-49cf-96cc-2cbe6e1d1436)

**Comment by pseudotensor** [comment]
```
docker pull vllm/vllm-openai:latest
docker stop qwen72bvll ; docker remove qwen72bvll
source ~/h2ogpt_ops/gr_exports.sh
docker run -d --restart=always \
    --runtime=nvidia \
    --gpus '"device=0,1,2,3"' \
    --shm-size=10.24gb \
    -p 5000:5000 \
        -e NCCL_IGNORE_DISABLED_P2P=1 \
    -v /etc/passwd:/etc/passwd:ro \
    -v /etc/group:/etc/group:ro \
    -u `id -u`:`id -g` \
    -e HUGGING_FACE_HUB_TOKEN=$HUGGING_FACE_HUB_TOKEN \
    -v "${HOME}"/.cache:$HOME/.cache/ -v "${HOME}"/.config:$HOME/.config/   -v "${HOME}"/.triton:$HOME/.triton/  \
    --network host \
    --name qwen72bvll \
    vllm/vllm-openai:latest \
        --port=5000 \
        --host=0.0.0.0 \
        --model=Qwen/Qwen2-VL-72B-Instruct \
        --tensor-parallel-size=4 \
        --seed 1234 \
        --trust-remote-code \
        --max-model-len=32768 \
        --max-num-batched-tokens 65536 \
        --max-log-len=100 \
        --download-dir=$HOME/.cache/huggingface/hub &>> logs.vllm_server.qwen72bvll.txt
```

leads to:

![image](https://github.com/user-attachments/assets/eba824d6-c928-4ec9-a925-1d34651b66f9)


using 0.6.0.post2.

**Comment by pseudotensor** [comment]
I can try to build off main

**Comment by pseudotensor** [comment]
Even a build off main b8747e8a7c318ab774862f94ccbdbba5b7d9dd4a after qwen2-vlm related fixes still fails:

```
NFO 09-24 13:01:18 api_server.py:164] Multiprocessing frontend to use ipc:///tmp/62289e81-c798-405a-b4f9-e8148f46337d for IPC Path.
INFO 09-24 13:01:18 api_server.py:177] Started engine process with PID 76
Traceback (most recent call last):
  File "/usr/local/lib/python3.12/dist-packages/transformers/models/auto/configuration_auto.py", line 993, in from_pretrained
    config_class = CONFIG_MAPPING[config_dict["model_type"]]
                   ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/transformers/models/auto/configuration_auto.py", line 695, in __getitem__
    raise KeyError(key)
KeyError: 'qwen2_vl'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/usr/local/lib/python3.12/dist-packages/vllm/entrypoints/openai/api_server.py", line 571, in <module>
    uvloop.run(run_server(args))
  File "/usr/local/lib/python3.12/dist-packages/uvloop/__init__.py", line 109, in run
    return __asyncio.run(
           ^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/asyncio/runners.py", line 194, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "uvloop/loop.pyx", line 1517, in uvloop.loop.Loop.run_until_complete
  File "/usr/local/lib/python3.12/dist-packages/uvloop/__init__.py", line 61, in wrapper
    return await main
           ^^^^^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/vllm/entrypoints/openai/api_server.py", line 538, in run_server
    async with build_async_engine_client(args) as engine_client:
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/contextlib.py", line 210, in __aenter__
    return await anext(self.gen)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/vllm/entrypoints/openai/api_server.py", line 105, in build_async_engine_client
    async with build_async_engine_client_from_engine_args(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/contextlib.py", line 210, in __aenter__
    return await anext(self.gen)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/vllm/entrypoints/openai/api_server.py", line 182, in build_async_engine_client_from_engine_args
    engine_config = engine_args.create_engine_config()
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/vllm/engine/arg_utils.py", line 874, in create_engine_config
    model_config = self.create_model_config()
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/vllm/engine/arg_utils.py", line 811, in create_model_config
    return ModelConfig(
           ^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/vllm/config.py", line 183, in __init__
    self.hf_config = get_config(self.model, trust_remote_code, revision,
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/vllm/transformers_utils/config.py", line 166, in get_config
    raise e
  File "/usr/local/lib/python3.12/dist-packages/vllm/transformers_utils/config.py", line 147, in get_config
    config = AutoConfig.from_pretrained(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/transformers/models/auto/configuration_auto.py", line 995, in from_pretrained
    raise ValueError(
ValueError: The checkpoint you are trying to load has model type `qwen2_vl` but Transformers does not recognize this architecture. This could be because of an issue with the checkpoint, or because your version of Transformers is out of date.
ubuntu@h2ogpt-a100-node-1:~/vllm$ 

```

**Comment by pseudotensor** [comment]
qwen2_vl github page says there's bug in transformers, so have to do:

```
pip install git+https://github.com/huggingface/transformers@21fac7abba2a37fae86106f87fcf9974fd1e3830 accelerate
```

will try.

**Comment by pseudotensor** [comment]
Modifying the requirements to use above avoided that problem.

```
(VllmWorkerProcess pid=211) WARNING 09-24 17:23:00 qwen2_vl.py:218] Current Qwen2-VL implementation has a bug with `vllm-flash-attn` inside vision module, so we use xformers backend instead. You can run `pip install flash-attn to use flash-attention backend.
```

See this though.

**Comment by pseudotensor** [comment]
So I would say vllm does *not* support this model OOTB with current transformers but the work around of installing that specific version of transformers does work.

**Comment by mgoin** [comment]
Unfortunately this is waiting on a release from Transformers, yes. They haven't pushed a release in a month :(

**Comment by xhluca** [comment]
Sorry for reviving this two months late.

I see that Qwen2-VL-72B-Instruct is in the list of supported models here: https://docs.vllm.ai/en/latest/models/supported_models.html#supported-vlms

Does that mean it is now official supported?
